velox/expression/VectorUdfTypeSystem.h

```diff
@@ -203,6 +203,24 @@ struct VectorReader {
     return decoded_.mayHaveNulls();
   }
 
+  // These functions can be used to check if any elements in a given row are
```

Comment:
I mean you are native speaker so i might be wrong. 
should it be if any element with in the container at a given row is null?

Reply:
I may be native, but I'm also prone to typos.

In this case though, I think both work (they both sound fine to me), as long as to be is conjugated correctly (are/is).

I tried Googling this, and there's a bunch of pedantic discussions about it on Stack Exchange, but I didn't see any conclusions.
https://english.stackexchange.com/questions/112996/any-followed-by-singular-or-plural-countable-nouns
https://ell.stackexchange.com/questions/1155/should-nouns-after-any-be-singular-or-plural